### PR TITLE
Fixing diffusers XL

### PR DESCRIPTION
**Pull request by Narsil**

### CODE REVIEW

**Comment by Narsil** [comment]
Tests are passing locally (in case those fail for whatever reason)

File: docker_images/diffusers/app/pipelines/text_to_image.py [hunk 1]
```diff
@@ -63,21 +57,13 @@ def __init__(self, model_id: str):
...
-            model_to_load = "kandinsky-community/kandinsky-2-1-prior"
-            self.ldm = KandinskyPipeline.from_pretrained(
-                model_id, use_auth_token=use_auth_token, **kwargs
-            )
-            self.prior = KandinskyPriorPipeline.from_pretrained(
-                model_to_load, use_auth_token=use_auth_token, **kwargs
-            )
         else:
-            self.ldm = DiffusionPipeline.from_pretrained(
+            self.ldm = AutoPipelineForTextToImage.from_pretrained(
```
Comment (patrickvonplaten) [inline docker_images/diffusers/app/pipelines/text_to_image.py:61]: AutoPipeline handles Kandinsky correctly

File: docker_images/diffusers/app/pipelines/text_to_image.py [hunk 2]
```diff
@@ -111,28 +95,9 @@ def __call__(self, inputs: str, **kwargs) -> "Image.Image":
     def _process_req(self, inputs, **kwargs):
         # only one image per prompt is supported
         kwargs["num_images_per_prompt"] = 1
-        if isinstance(self.ldm, (StableDiffusionPipeline, AltDiffusionPipeline)):
+        self.is_karras_compatible:
             if "num_inference_steps" not in kwargs:
                 kwargs["num_inference_steps"] = 25
```
Comment (patrickvonplaten) [inline docker_images/diffusers/app/pipelines/text_to_image.py:100]:
```suggestion
        if self.is_karras_compatible and "num_inference_steps" not in kwargs:
            kwargs["num_inference_steps"] = 25
```

**Comment by Narsil** [comment]
Everything looks fine in the tests, merging.

**Comment by Narsil** [comment]
Don't hesitate to merge if you need to btw (without me approving) everything is CI/CD

**Comment by patrickvonplaten** [comment]
Yeah thanks!

**Pull request merged**